src/main/java/org/jabref/gui/customizefields/CustomizeGeneralFieldsDialogView.java

```diff
@@ -0,0 +1,72 @@
+package org.jabref.gui.customizefields;
+
+import javax.inject.Inject;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.TextArea;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.help.HelpAction;
+import org.jabref.gui.util.BaseDialog;
+import org.jabref.gui.util.ControlHelper;
+import org.jabref.logic.help.HelpFile;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.preferences.PreferencesService;
+
+import com.airhacks.afterburner.views.ViewLoader;
+
+public class CustomizeGeneralFieldsDialogView extends BaseDialog<Void> {
+
+    @FXML private ButtonType resetButton;
+    @FXML private ButtonType helpButton;
+    @FXML private ButtonType okButton;
+    @FXML private ButtonType cancelButton;
```

Comment:
If you don't use the cancel button further in your code, you can remove it here and in the fxml remove the` fx:id` from it.